modules/Microsoft.Network/dnsResolvers/.bicep/nested_roleAssignments.bicep

```diff
@@ -0,0 +1,55 @@
+@sys.description('Required. The IDs of the principals to assign the role to.')
```

Comment:
There are a couple parameters missing. Please check e.g the [Logic App module](https://github.com/Azure/ResourceModules/blob/main/modules/Microsoft.Logic/workflows/.bicep/nested_roleAssignments.bicep) for reference.

Also requires an update of the invoking parent bicep.

Reply:
Resolved in latest commit